things/database.py

```diff
@@ -76,6 +79,7 @@
 # Date Columns
 # --------------------------------------------------
 
+DATE_START = "startDate"
```

Comment:
Rule: keep alphabetical order. It makes reading easier.

Reply:
ok, done